### DISCUSSION TITLE
put examples in docs

### DISCUSSION THREAD

**Discussion by j042** [tataratat/splinepy#413] [General]
@clemens-fricke

**Comment by clemens-fricke** [comment]
The main problem currently is that the plotting via vedo is not working in headless mode. I have briefly looked into this in the `tataratat/gustaf` repo, but have not found a solution yet.

Solving this problem with headless mode is also something we should do in gustaf since that is the visualizing pre-backend we use. Otherwise, I have already readied the file `examples/run_all_examples.py` to be able to call and run all examples and notify the calling process if any example fails. I would guess that solving the problem with headless mode will enable us to call the examples in addition to the tests during CI/CD. 

I am not sure if we could/want to include the examples in the `pytest` suite or automatically call them during these tests.

**Comment by jzwar** [comment]
As I don't want to open an issue for this:
There is some typesetting issues in steffi's part of the documentation. Some of the lower indices are not set properly and also there is an issue with the brackets used in the docs. We should update that in the next iteration
@j042

**Reply by j042** [reply]
good catch! @markriegler - can you take a look at this?

**Reply by markriegler** [reply]
I can look into it
